hackathon_site/registration/test_admin.py

```diff
@@ -0,0 +1,62 @@
+import csv
+import io
+
+from django.test import TestCase
+from django.urls import reverse
+from rest_framework import status
+
+from hackathon_site.tests import SetupUserMixin
+from registration.admin import ApplicationAdmin
+
+
+class ExportCsvAdminTestCase(SetupUserMixin, TestCase):
+    """
+    Test that CSV exportation in the Registration/Application tab works as expected.
+    """
+
+    def setUp(self):
+        super().setUp()
+        self.application_view = reverse("admin:registration_application_changelist")
+        self.export_fields = [field[0] for field in ApplicationAdmin.export_fields]
```

Comment:
Would it be better to hard code these in the test rather than importing them? There may be an edge case where someone typos the class attribute and this wouldn't catch it, but that may be far enough of an edge case that we don't care.